.drone.yml

```diff
@@ -61,6 +61,48 @@ pipeline:
         TEST_SUITE: phpunit
         PHP_VERSION: 7.1
 
+  configure-server:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud
+      - php occ a:l
+      - php occ a:e testing
+      - php occ a:e encryption
+      - php occ encryption:enable
+      - php occ encryption:select-encryption-type ${ENCRYPTION_TYPE} --yes
+      - php occ a:l
+      - php occ config:system:set trusted_domains 1 --value=server
+      - php occ config:system:set trusted_domains 2 --value=federated
+      - php occ log:manage --level 0
+      - php occ config:list
+      - chown -R www-data. /var/www/owncloud
+    when:
+      matrix:
+        PREPARE_ACCEPTANCE: true
+
+  webui-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - BROWSER=chrome
+      - SELENIUM_HOST=selenium
+      - SRV_HOST_NAME=server
+      - SRV_HOST_PORT=80
+      - REMOTE_FED_SRV_HOST_NAME=federated
+      - REMOTE_FED_SRV_HOST_PORT=80
+      - SKELETON_DIR=/var/www/owncloud/tests/acceptance/webUISkeleton
+      - SELENIUM_PORT=4444
+      - PLATFORM=Linux
+      - MAILHOG_HOST=email
+    commands:
+      - cd /var/www/owncloud
+      - chmod 777 /var/www/owncloud/tests/acceptance/filesForUpload -R
+      - bash tests/travis/start_ui_tests.sh --remote --all-suites
```

Comment:
Now that https://github.com/owncloud/core/pull/31808 is in core, it should be possible to skip core tests that are different or not relevant for encryption by adding:
```
--tags '~@skipOnEncryption'
```
in a similar way to the ``.drone.yml`` in the ``user_ldap`` app.

Reply:
will push that later tonight / tomorrow - once the tags are in stable10 / master